frappe/integrations/doctype/social_login_key/social_login_key.py

```diff
@@ -56,7 +56,7 @@ class SocialLoginKey(Document):
 		redirect_url: DF.Data | None
 		sign_ups: DF.Literal["", "Allow", "Deny"]
 		social_login_provider: DF.Literal[
-			"Custom", "Facebook", "Frappe", "GitHub", "Google", "Office 365", "Salesforce", "fairlogin"
```

Comment:
Did you edit this manually? (I don't see the JSON changes)

You should use the doctype editor for this

Reply:
Yes previously I added it manually, but in the recent commit I made changes through erpnext platform from the doctype field settings, you can see the changes in the json file.